src/tds/login.c

```diff
@@ -418,6 +418,12 @@ tds_setup_connection(TDSSOCKET *tds, TDSLOGIN *login, bool set_db, bool set_spid
 	char *str;
 	int len;
 	bool parse_results = false;
+	bool is_anywhere
```

Comment:
maybe adding also a line
```c
const char *const product_name = tds->conn->product_name != NULL ? tds->conn->product_name : "";
```
and reuse it could help simplifying the code.